src/oatpp/network/tcp/client/ConnectionProvider.cpp

```diff
@@ -282,7 +282,7 @@ oatpp::async::CoroutineStarterForResult<const provider::ResourceHandle<data::str
     Action doConnect() {
       errno = 0;
 
-      auto res = connect(m_clientHandle, m_currentResult->ai_addr, static_cast<int>(m_currentResult->ai_addrlen));
+      auto res = connect(m_clientHandle, m_currentResult->ai_addr, static_cast<socklen_t>(m_currentResult->ai_addrlen));
```

Comment:
Let's use a newly introduced `v_sock_size`

Reply:
merged master that has this